### ISSUE TITLE
Repository is unusable

### ISSUE THREAD

**Issue by stemy2** [firsttris/repository.sendtokodi.python3#5]
Hello,

I can't install the addon because kodi says it can't access to the repository. The logs: https://paste.kodi.tv/ihedivazik

**Comment by nullket** [comment]
Hello, 

yes unfortunately we did not catch that issue before the release of nexus where a scheme we used in our repo is now deprecated. It is fixed but the repo on your computer can not update itself. Please uninstall the addon, the repo (!) an reinstall the repo from the zip linked in the readme (must be >= 1.0.0 for the python3 version)

**Comment by stemy2** [comment]
The zip linked in the readme is precisely the one that bothers me.

**Comment by nullket** [comment]
Your log states
```
repository.sendtokodi.python3 v0.0.1 installed
```

Are you sure  you have **both** the addon and the repo uninstalled before you tried to install v1.0.0 of the repo?

The readme links this file, just to be sure, you are using this one right? https://github.com/firsttris/repository.sendtokodi.python3/raw/master/repository.sendtokodi.python3/repository.sendtokodi.python3-1.0.0.zip

This new version does not contain the version 0.0.1 anywhere. In every file of the zip it states 1.0.0. Therefore based on your log I assume you still have the old version.

**Comment by stemy2** [comment]
The repo mentioned in the readme is the v.0.0.1, not the v1.0.0.

**Comment by nullket** [comment]
The readme that shows on this GitHub repo https://github.com/firsttris/plugin.video.sendtokodi links to the 1.0.0 version where it says: „[Download repo for kodi 19+](https://github.com/firsttris/repository.sendtokodi.python3/raw/master/repository.sendtokodi.python3/repository.sendtokodi.python3-1.0.0.zip)“

The file behind this url was changed over a week ago. Are you maybe still having the old file from over a week ago and thought the file behind the link did not change? 

Maybe I am missing something? Is there another link which points to the old 0.0.1 version somewhere?

**Comment by nullket** [comment]
Ah dang it. Now I see. The readme of the repo itself is still linking a wrong file. Thanks for printing out!

**Comment by nullket** [comment]
I am closing this as I just fixed the link in the other readme and it should be resolved once you install the new version. Please report if that is really the case for you and if not we reopen the issue.

**Comment by stemy2** [comment]
The new repository doesn't work either, but for another reason: i can't even install it because kodi says its structure is invalid.

**Comment by nullket** [comment]
Can you please provide a log from the installation process?

**Comment by stemy2** [comment]
The logs are not very helpful, but there it is:
https://paste.kodi.tv/efomufehad

**Comment by nullket** [comment]
- According to the log you have still 0.0.1 installed. Please make sure to remove all traces (uninstall addon, repo and maybe even restart) before reinstalling.
- According to the log you have the addon in version v100.0.0. This is usually our placeholder for development. Have you maybe cloned the sendtokodi repo and not installed it via the repo at one time? 

From your log
```
2023-03-19 11:32:50.615 T:1701     info <general>: CAddonMgr::FindAddons: plugin.video.sendtokodi v100.0.0 installed
....
2023-03-19 11:32:50.615 T:1701     info <general>: CAddonMgr::FindAddons: repository.sendtokodi.python3 v0.0.1 installed
```

**Comment by stemy2** [comment]
This is strange beause i carefully ininstalled the v0.0.1 repo and it is not present anymore in my repo list. Since kodi says that the file structure is invalid, i'm not sure the problem comes from my side, i think you should try the repo by yourself.

**Comment by nullket** [comment]
> Since kodi says that the file structure is invalid, i'm not sure the problem comes from my side

Your log says `Repository add-on repository.sendtokodi.python3 uses old schema definition for the repository extension point`. This was only true for the 0.0.1 version of the repo and is fixed in the 1.0.0 version. Your log literally says `repository.sendtokodi.python3 v0.0.1 installed` as long as this is the case I am fairly certain the issue is on your side ;) 

As mentioned you also have the addon in v100.0.0 installed which indicates that you not just installed sendtokodi from the repo and might need to dig a bit deeper for cleaning.

Please uninstall the addon and the repo, restart kodi and check the log for any traces of anything related  with sendtokodi. If there are still traces sendtokodi or its repo you need to fix those first. 


An for the sake of completeness I just setup a fresh kodi installation, installed the repo from zip and then the addon from the repo:
```log
2023-03-19 13:47:16.945 T:7        info <general>: JSONRPC Server: Successfully initialized
2023-03-19 13:47:16.945 T:7        info <CWebserver[8888]>: Started
2023-03-19 13:47:16.945 T:7        info <general>: CWSDiscoveryListenerUDP::Start - Started
2023-03-19 13:47:16.977 T:7        info <general>: GL: Maximum texture width: 32768
2023-03-19 13:47:17.869 T:32       info <general>: Skipped 1 duplicate messages..
2023-03-19 13:47:17.869 T:32       info <general>: JSONRPC Server: New connection added
2023-03-19 13:47:17.894 T:7        info <general>: Loading skin file: MyVideoNav.xml, load type: KEEP_IN_MEMORY
2023-03-19 13:47:31.310 T:7        info <general>: Loading skin file: Settings.xml, load type: KEEP_IN_MEMORY
2023-03-19 13:47:33.395 T:7        info <general>: Loading skin file: AddonBrowser.xml, load type: KEEP_IN_MEMORY
2023-03-19 13:47:39.013 T:7        info <general>: Loading skin file: FileBrowser.xml, load type: KEEP_IN_MEMORY
2023-03-19 13:47:43.911 T:7     warning <general>: CGUIWindowManager - HandleAction - ignoring action 103, because topmost modal dialog closing animation is running
# Installing from zip
2023-03-19 13:47:44.124 T:17       info <general>: CAddonMgr::FindAddon: repository.sendtokodi.python3 v1.0.0 installed
# Installing from the repo
2023-03-19 13:48:03.927 T:18       info <general>: CAddonMgr::FindAddon: plugin.video.sendtokodi v0.9.423 installed

```

**Comment by stemy2** [comment]
I rebooted and now, i was able to install the repo.
